### DISCUSSION TITLE
Classic Start

### DISCUSSION THREAD

**Discussion by RW4x4van** [Open-Shell/Open-Shell-Menu#1965] [General]
Hello,....A recent update from Chrome and Microsoft has changed something and I don't' know what it is...
When I right click to highlight a line on a website, that I'm already logged into, a box appears stating "You need to be logged in"
Would "Classic Start" cause this........TIA.....![log in](https://github.com/user-attachments/assets/7cd84f28-4361-42a1-9cf0-7b67efc37f5f)

**Comment by ge0rdi** [comment]
That's not related to `Open-Shell` as it has nothing to do with web browsers.

**Reply by RW4x4van** [reply]
Thanks, I didn't think so
